README.md

```diff
@@ -4,14 +4,20 @@ This is a [Stytch](https://stytch.com) + [Next.js](https://nextjs.org/) project
 
 In this repo, we have two sample auth flows:
 
-- SDK integration: This flow uses Stytch's React component to create a login and signup flow using email magic links. Other libraries used include Stytch's node client library for authentication and with-iron-sessions for session management.
-- API integration: This flow uses a custom UI with Stytch's backend API for SMS OTP authentication. Both Stytch's node client library and with-iron-session are also used.
+- SDK integration: This flow uses Stytch's React component to create a login and sign-up flow using [Email Magic Links](https://stytch.com/docs/api/send-by-email).
+- API integration: This flow uses a custom UI with Stytch's backend API for [Onetime Passcodes(OTP) via SMS](https://stytch.com/docs/api/sms-otp-overview) authentication.
 
-# Getting Started
+Both flows use Stytch's [Node client library](https://github.com/stytchauth/stytch-node) and `with-iron-session` for session management.
 
-### Setting up Stytch
+<!-- If you'd like to deploy with [Vercel](https://vercel.com/), just add `https://*.vercel.app:3000` as both a login and sign-up re-direct URL to your [Stytch Dashboard](https://stytch.com/dashboard/redirect-urls) and then click Deploy below! You'll be asked to login to your Vercel account, then we'll walk you through each step to get you up-and-running in no time. -->
 
-After signing up for Stytch, you'll need your project's ID, secret, and public token. You can find these in the [API keys tab](https://stytch.com/dashboard/api-keys).
+<!-- [![Deploy with Vercel](TODO) -->
```

Comment:
Keeping this hidden until I can confirm the correct deploy button config. Without this PR being on `main`, I can't test deploying with Vercel as that pulls against `main`.